### PR TITLE
Task/g109 optional duration for mosaic transaction

### PR DESCRIPTION
**Pull request by rg911**
Issue: #109 

Changed to make `Duration` optional on MosaicDefinition transaction

### CODE REVIEW

File: test/core/utils/TransactionMapping.spec.ts [hunk 1]
```diff
@@ -191,8 +191,33 @@ describe('TransactionMapping', () => {
 
         const transaction = TransactionMapping.createFromPayload(signedTransaction.payload) as MosaicDefinitionTransaction;
 
-        expect(transaction.mosaicProperties.duration.lower).to.be.equal(1000);
-        expect(transaction.mosaicProperties.duration.higher).to.be.equal(0);
+        expect(transaction.mosaicProperties.duration!.lower).to.be.equal(1000);
+        expect(transaction.mosaicProperties.duration! .higher).to.be.equal(0);
```
Comment (evias) [inline test/core/utils/TransactionMapping.spec.ts]: remove space before `.higher`

File: tslint.json [hunk 2]
```diff
@@ -139,6 +139,6 @@
     "use-pipe-transform-interface": true,
     "component-class-suffix": true,
     "directive-class-suffix": true,
-    "invoke-injectable": true
+    "invoke-injectable": true,
```
Comment (evias) [inline tslint.json]: why adding this comma ? this is not standard JSON (your editor may sometimes require "trailing-comma" but valid JSON is without.

Comment (rg911) [reply tslint.json]: 👍

File: test/core/utils/TransactionMapping.spec.ts (hunk 1 above)
Comment (rg911) [reply test/core/utils/TransactionMapping.spec.ts]: done

**Pull request merged**